recipes/omniscidb-cuda/yum_requirements.txt

```diff
@@ -0,0 +1,2 @@
+# cudatoolkit has hardcoded link to /usr/bin/perl under linux; cannot use conda-perl
+perl
```

Comment:
Could you please elaborate on this workaround a bit?

cc @jjhelmus (for awareness)

Reply:
This can probably be eliminated after we drop using cudatoolkit-dev conda package (that runs the cudatoolkit installer when one installer script contains the hardcoded link to system `perl`).

Reply:
Removed perl.